frontend/packages/dev-console/src/components/helm/HelmInstallPage.tsx

```diff
@@ -1,41 +1,82 @@
 import * as React from 'react';
 import * as yup from 'yup';
+import { safeDump, safeLoad } from 'js-yaml';
 import { RouteComponentProps } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
-import { PageHeading, history } from '@console/internal/components/utils';
+import { PageHeading, history, LoadingBox } from '@console/internal/components/utils';
 import { coFetchJSON } from '@console/internal/co-fetch';
 import { Formik } from 'formik';
 import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
 import { nameValidationSchema } from '../import/validation-schema';
 import HelmInstallForm from './HelmInstallForm';
+import { HelmChart } from './helm-types';
 
 export type HelmInstallPageProps = RouteComponentProps<{ ns?: string }>;
 
 export type HelmInstallFormData = {
   releaseName: string;
+  chartValuesYAML: string;
 };
 
 export const HelmInstallPage: React.FunctionComponent<HelmInstallPageProps> = ({ location }) => {
   const searchParams = new URLSearchParams(location.search);
   const chartURL = decodeURIComponent(searchParams.get('chartURL'));
+  const chartName = searchParams.get('chartName');
   const preselectedNamespace = searchParams.get('preselected-ns');
+  const [chartData, setChartData] = React.useState<HelmChart>();
+  const [chartDataLoaded, setChartDataLoaded] = React.useState(false);
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const fetchHelmReleases = async () => {
+      let res: HelmChart;
+      try {
+        res = await coFetchJSON(`/api/helm/chart?urls=${chartURL}`);
+      } catch {
+        if (ignore) return;
+      }
+      if (ignore) return;
+
+      setChartData(res);
+      setChartDataLoaded(true);
+    };
+
+    fetchHelmReleases();
+
+    return () => {
+      ignore = true;
+    };
+  }, [chartURL]);
 
   const initialValues: HelmInstallFormData = {
-    releaseName: '',
+    releaseName: chartName ? `${chartName}-defaulted` : '',
```

Comment:
I know the screenshot in design has a suffix `-defaulted` but was this really the intention? Why not default to simply the chart name itself? Was this confirmed with UX or assumed based on screenshot?

cc @Veethika  @serenamarie125

Reply:
I just followed what the UX design doc had in the screenshot assuming that's what the ask was. @serenamarie125 @Veethika can you confirm?

Reply:
Since it wasn't explicitly called out, i feel like it was just an artifact of design filling in arbitrary data.

I don't know how a suffix of `-defaulted` is better than simply the chart name itself.

Reply:
Updated to have just the chart name as default pre-populated release name.